src/stories/Flexible_GridGalleryCards.stories.js

```diff
@@ -0,0 +1,79 @@
+import FlexibleGridGalleryCards from '@/lib-components/Flexible/GridGalleryCards'
+
+export default {
+  title: 'FLEXIBLE / Grid Gallery Cards',
+  component: FlexibleGridGalleryCards,
+}
+
+const mock = {
+  id: '40361',
+  subtitle: 'Flexible Grid Gallery Cards',
```

Comment:
I just looked at the graphQL query https://github.com/UCLALibrary/library-website-nuxt/blob/main/gql/fragments/BlockGridGalleryCardsFragment.gql, we can safely remove subtitle and summary here